core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -53,7 +53,7 @@ private[op] trait TransmogrifierDefaults {
   val NullString: String = OpVectorColumnMetadata.NullString
   val OtherString: String = OpVectorColumnMetadata.OtherString
   val DefaultNumOfFeatures: Int = 512
-  val MaxNumOfFeatures: Int = 16384
+  val MaxNumOfFeatures: Int = 1638400000
```

Comment:
the above number is 2^14 please choose a number in bytes

Reply:
i think we don't need to change this number, because i lifted this limit in #477